TidepoolOnboardingKitUI/Views/OnboardingSectionPagesView.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  OnboardingSectionPagesView.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Darin Krauss on 3/4/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+import SwiftUI
+
+struct OnboardingPage {
+    let title: String
+    let view: AnyView
+
+    init<T: View>(title: String, view: T) {
+        self.title = title
+        self.view = AnyView(view)
+    }
+}
+
+struct OnboardingSectionPages {
+    let section: OnboardingSection
+    let pages: [OnboardingPage]
+
+    init(section: OnboardingSection, pages: [OnboardingPage]) {
+        self.section = section
+        self.pages = pages
+    }
+
+    var count: Int { pages.count }
+
+    func title(for pageNumber: Int) -> String { pages[pageNumber - 1].title }
```

Comment:
[nit] These will unceremoniously crash if `pageNumber < 1 || pageNumber > count`, which is probably fine because it would constitute a programming error (I think?).  But this class doesn't really "know" that.  

This note is just to ask for you to consider avoiding crashing, and if you decide that it's okay, I'm fine with it.

Reply:
This class goes away entirely in my next chunk of code. I decided this implementation was too heavy handed. In any case, this issue is now moot. 😄